legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -328,6 +328,28 @@
                {% endif %}
                <div class="pt-2">{{report_date_time}}</div>
             </td>
+         {% elif header.name == 'noticeOfWithdrawal' %}
+            <td class="col-33">
+               {% if not business or business.identifier.startswith('T') %}
+                  <div class="bold">Filed Date and Time:</div>
+               {% else %}
+                  <div class="bold">Incorporation Number:</div>
```

Comment:
You're not going to have an incorporation number (see also line 347) if the business identified starts with a T. Did Jacqueline want the T number displayed anyway?

See pink NOTES box:
![image](https://github.com/user-attachments/assets/4932fbf4-2610-4bbd-8237-169809bdd4a6)

Reply:
The 'T' number won't be rendered, only if it's a business and identifier not starts with 'T' will render the incorporation number section. As you can see the logic in line 333 and 334.

Reply:
Will double-check and verify this when doing the other ticket when supporting NoW for temporary businesses.

Reply:
Oh, I saw the "not" but missed the "or". I think that logic is OK.

So, for a T business, the display is this?
- Filed Date and Time:
- Recognition Date and Time: (ie, effective) **maybe this should be a different label?**
- Retrieved Date and Time: (ie, report)

If OK then you can resolve this comment.

Reply:
Yes, these 3 items for T business, follows the design. and will be verified later. 
![image](https://github.com/user-attachments/assets/e8d7df5b-5f01-409d-83cd-b829beb88177)

and I've left a question to Jacqueline in the ticket for the expected value for the 'Recognition Date and Time', we will know when she's back. And I will update the details in the other tickets when it's clarified. 😃